src/reconstruction/refinement/addReactionGEM.m

```diff
@@ -69,6 +69,8 @@
 end
 
 nMet = length(model.mets)+1;
+orignMets = numel(model.mets);

```

Comment:
orignMets is the number of mets at the start. nMets is modified during the function, so we need 2 separate values because to properly update the remaining met fields, we need to know what the original size was.